storehaus-cassandra/src/main/scala/com/twitter/storehaus/cassandra/cql/CQLCassandraStoreTupleValues.scala

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2014 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.storehaus.cassandra.cql
+
+import com.datastax.driver.core.{BatchStatement, ConsistencyLevel, ResultSet, Row, SimpleStatement, Statement}
+import com.datastax.driver.core.policies.{Policies, RoundRobinPolicy, ReconnectionPolicy, RetryPolicy, TokenAwarePolicy}
+import com.datastax.driver.core.querybuilder.{BuiltStatement, Insert, QueryBuilder, Update}
+import com.twitter.concurrent.Spool
+import com.twitter.storehaus.{IterableStore, QueryableStore, ReadableStore, ReadableStoreProxy, Store, WithPutTtl}
+import com.twitter.util.{Await, Closable, Future, Duration, FuturePool, Promise, Try, Throw, Return}
+import com.websudos.phantom.CassandraPrimitive
+import java.util.concurrent.Executors
+import shapeless._
+import shapeless.ops.hlist.{Mapper, ToList, Tupler}
+import shapeless.ops.traversable.FromTraversable
+import shapeless.UnaryTCConstraint.*->*
+import shapeless.syntax.std.product._
+import syntax.std.traversable._
```

Comment:
`shapeless.syntax.std.traversable._` would be better for consistency and clarity.